docs/libertyDev.md

```diff
@@ -141,4 +141,5 @@ These can also be specified as command line parameters in addition to the ones i
 | --------- | ----- | ----- | ----------- | -------- |
 | container | boolean | 3.1-M1 (tech preview) | If set to `true`, run the server in the container specified by the `dockerfile` parameter. Setting this to `true` and using the `libertyDev` task is equivalent to using the `libertyDevc` task. The default value is `false` when the `libertyDev` task is used, and `true` when the `libertyDevc` task is used. | No |
 | dockerRunOpts | String | 3.1-M1 (tech preview) | Specifies options to add to the `docker run` command when using dev mode to launch your server in a container. For example, `-e key=value` is recognized by `docker run` to define an environment variable with the name `key` and value `value`. Setting this parameter overrides the `container` parameter to `true`. | No |
-| dockerfile | File | 3.1-M1 (tech preview) | Location of a Dockerfile to be used by dev mode to build the container that runs your server and to specify the context used to build the container image. The default location is `Dockerfile` in the project root. Setting this parameter overrides the `container` parameter to `true`. | No |
\ No newline at end of file
+| dockerfile | File | 3.1-M1 (tech preview) | Location of a Dockerfile to be used by dev mode to build the container that runs your server and to specify the context used to build the container image. The default location is `Dockerfile` in the project root. Setting this parameter overrides the `container` parameter to `true`. | No |
+| dockerBuildTimeout | integer | 3.1-M2 (tech preview) | Maximum time to wait (in seconds) for the completion of the docker operation to build the container. The value must be an integer greater than 0. The default value is `60` seconds. | No |
```

Comment:
Same as https://github.com/OpenLiberty/ci.maven/pull/907#discussion_r474249405

Reply:
Fixed in both places.